samples/drivers/led_max7219/src/main.c

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
do we really need yet another copy of this sample for another driver? This seems to the same as led_lp3943. Either unify and make existing sample work for new driver or provide something that is new.